### PR TITLE
add method for querying views without owner

### PR DESCRIPTION
**Pull request by stropitek**
- chore: remove no-explicity-any rule
- feat: add getView to Roc

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -4,5 +4,6 @@ env:
 parserOptions:
   project: tsconfig.base.json
 rules:
-  camelcase: off
-  '@typescript-eslint/no-dynamic-delete': off
+  camelcase: 0
+  '@typescript-eslint/no-dynamic-delete': 0
+  '@typescript-eslint/no-explicit-any': 0
```
Comment (targos) [inline .eslintrc.yml:9]:
```suggestion
  camelcase: off
  '@typescript-eslint/no-dynamic-delete': off
  '@typescript-eslint/no-explicit-any': off
```

I prefer explicitness of the words

**Review by targos — APPROVED** [review]
lgtm if it works

File: .eslintrc.yml (hunk 1 above)
Comment (stropitek) [reply .eslintrc.yml:9]: I was working around this:
image

Putting "off" in quotes also removes the error... I don't know which extension if any is causing this error. Seems like a false positive.

Comment (targos) [reply .eslintrc.yml]: yeah, it's weird. `off` is a string

**Pull request merged**